lms/djangoapps/branding/tests/test_page.py

```diff
@@ -208,7 +207,7 @@ def test_course_discovery_off(self):
         self.assertNotIn('Search for a course', response.content)
 
         # check the /courses view
-        response = self.client.get(reverse('branding.views.courses'))
+        response = self.client.get(reverse('courses'))
```

Comment:
Curious: Why is `branding.views.` no longer needed? Is the test class aware of the Django app in which it's being run? Otherwise, seems like `courses` is quite non-specific.

Reply:
"courses" has always been the canonical name of that URL as per `lms/urls.py`.  This is just an instance of looking up a URL by its view function Python path instead of assigned name which we missed in the cleanup about a month ago.